js/runner.go

```diff
@@ -164,6 +255,14 @@ func (r *Runner) newVU(id int64, samplesOut chan<- stats.SampleContainer) (*VU,
 		Blacklist: r.Bundle.Options.BlacklistIPs,
 		Hosts:     r.Bundle.Options.Hosts,
 	}
+	if r.Bundle.Options.ClientIpRange.Valid {
+		ippool := GetIpPool(r.Bundle.Options.ClientIpRange.ValueOrZero())
```

Comment:
GetIpPool() should be called once in SetOptions, but I am not familiar with that code area to support it on both CLI and Config file...